### PR TITLE
Adding some of the software mentioned in issues

### PR DESCRIPTION
**Pull request by cpdevelops**
Hello there! It has been a while since I contributed to this project so I decided I was going to clean up the issues folder a little bit. 

This pull request completes issues #1087 #1140 and #1138 

Something to note is that I named one commit "add filerun + wondercms", that was a mistake. I am waiting for a response from the original creator of the issue to provide the license and language. 

Thank you for looking at this PR and I am available to make any necessary changes.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
HackMD |   * [HackMD](https://www.hackmd.io) - Realtime collaborative markdown notes on all platforms. `MIT`  `Node.js` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by cpdevelops** [comment]
I forgot to add the source code URL to the HackMD listing and I believe that is what tripped @danger-awesome-Selfhosted up. I added that URL so the issue should hopefully be fixed now.

File: README.md [hunk 1]
```diff
@@ -474,6 +474,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Textpattern](http://textpattern.com/) - A flexible, elegant and easy-to-use CMS. ([Demo](http://textpattern.co/demo), [Source Code](https://github.com/textpattern/textpattern)) `GPLv2` `PHP`
   * [TYPO3](https://typo3.org/) - A powerful and advanced CMS with a large community. ([Source Code](https://github.com/TYPO3/TYPO3.CMS)) `GPLv2` `PHP`
   * [Umbraco](https://umbraco.com/) - The friendly CMS. Free and open source with an amazing community. ([Source Code](https://github.com/umbraco/Umbraco-CMS)) `MIT` `.NET`
+  * [WonderCMS](http://www.wondercms.com) - WonderCMS is the smallest flat file CMS since 2008. ([Demo](https://www.wondercms.com/demo), [Source Code](https://github.com/robiso/wondercms)) `MIT` `PHP and jQuery`
```
Comment (nodiscc) [inline README.md]: `PHP` only (javascript is only required on the clilent side and is a give for a web application anyway)

File: README.md [hunk 2]
```diff
@@ -563,6 +564,7 @@ See https://github.com/n1trux/awesome-sysadmin#identity-management
   * [RSS Monster](https://github.com/pietheinstrengholt/rssmonster) - RSS Monster is an easy to use web-based RSS aggregator and reader compatible with the Fever API, created as an alternative for Google Reader. `MIT` `PHP`
   * [Screaming Liquid Tiger](https://github.com/herrbischoff/screaming-liquid-tiger) - Simple script to automatically generate valid RSS and Atom feeds from a list of media files in the same folder. `MIT` `PHP`
   * [Selfoss](http://selfoss.aditu.de/) - The new multipurpose rss reader, live stream, mashup, aggregation web application. ([Source Code](https://github.com/SSilence/selfoss)) `AGPLv3` `PHP`
+  * [Sismics Reader](https://github.com/sismics/reader) - Free and open source feeds reader, including all major Google Reader features. `GPL2` `Java`
```
Comment (nodiscc) [inline README.md]: `GPLv2`

File: README.md [hunk 3]
```diff
@@ -849,6 +851,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 *See also [Wikis](#wikis)*
 
   * [dillinger](http://dillinger.io/) - The last Markdown editor, ever. ([Source Code](https://github.com/joemccann/dillinger/)) `MIT` `Nodejs`
+  * [HackMD](https://www.hackmd.io) - Realtime collaborative markdown notes on all platforms. ([Source Code](https://github.com/hackmdio/hackmd/)) `MIT`  `Node.js`
```
Comment (nodiscc) [inline README.md]: Please remove the double space and use `Nodejs` (no dot)

**Comment by nodiscc** [comment]
@cpdevelops Some changes are needed; thanks for the pull request

**Comment by nodiscc** [comment]
@cpdevelops are you still interested in merging this? If yes, please address the requested changes and I will gladly merge it. Else this can be closed.

**Comment by cpdevelops** [comment]
Oh, sorry! Forgot about this pull request due to different things clogging up my time developing. Let me fix these issues and get it merged.

**Comment by cpdevelops** [comment]
These commits fix all the changes you requested with no merge conflicts @nodiscc

**Comment by robiso** [comment]
Thank you @cpdevelops for taking the time and adding changes.
@nodiscc additional shout out and thanks to you for making this possible.

Excellent work guys :+1:.

**Pull request merged**